antarest/study/storage/rawstudy/model/filesystem/root/output/simulation/ts_generator/ts_generator.py

```diff
@@ -56,17 +56,18 @@ def build(self) -> TREE:
 
 class OutputSimulationTsGenerator(FolderNode):
     def build(self) -> TREE:
-        children: TREE = {
-            "hydro": OutputSimulationTsGeneratorCustomMatrixList(
+        children: TREE = {}
```

Comment:
Cette partie du code n'est pas couverte pas les tests unitaires.

Reply:
I believe it's because the ts-generator part is not supported yet in AntaresWeb.